jetstream/first-run-easy-as-one-click-ver-1.toml

```diff
@@ -0,0 +1,44 @@
+[experiment]
+start_date = "2022-11-29"
+end_date = "2023-01-03"
+enrollment_period = 15
+
+[metrics]
+weekly = ['is_pinned', 'imported_bookmarks', 'imported_logins', 'imported_history']
+
+overall = ['is_pinned', 'imported_bookmarks', 'imported_logins', 'imported_history']
+
+## Imported Bookmarks
+[metrics.imported_bookmarks]
+data_source = "main"
+select_expression = "COUNTIF(COALESCE((SELECT SUM(`mozfun`.hist.extract(x.value).sum) FROM UNNEST(payload.keyed_histograms.fx_migration_bookmarks_quantity) x WHERE x.key != 'firefox'), 0) > 0)"
+type = "scalar"
+frienly_name = "Imported Bookmarks"
+description = "Count of clients who imported bookmarks"
+category = "Device Migration"
+
+[metrics.imported_bookmarks.statistics.binomial]
+
+
+## Imported Logins
+[metrics.imported_logins]
+data_source = "main"
+select_expression = "COUNTIF(COALESCE((SELECT SUM(`mozfun`.hist.extract(x.value).sum) FROM UNNEST(payload.keyed_histograms.fx_migration_logins_quantity) x WHERE x.key != 'firefox'), 0) > 0)"
+type = "scalar"
+friendly_name = "Imported Logins"
+description = "Count of clients who imported logins"
+category = "Device Migration"
+
+[metrics.imported_logins.statistics.binomial]
+
+
+## Imported History
+[metrics.imported_history]
+data_source = "main"
+select_expression = "COUNTIF(COALESCE((SELECT SUM(`mozfun`.hist.extract(x.value).sum) FROM UNNEST(payload.keyed_histograms.fx_migration_history_quantity) x WHERE x.key != 'firefox'), 0) > 0)"
+type = "scalar"
+friendly_name = "Imported History"
+description = "Count of clients who imported history"
+category = "Device Migration"
+[metrics.imported_history.statistics.binomial]
+
```

Comment:
```suggestion
```

No need to redefine the metrics here, you can just reference their slug

Reply:
their slug is their name? for example, referencing in the metrics section by name should be enough?

```
[metrics]
weekly = ['is_pinned', 'imported_bookmarks', 'imported_logins', 'imported_history']
overall = ['is_pinned', 'imported_bookmarks', 'imported_logins', 'imported_history']
```

Reply:
yes, that's how you'd reference them